Index: source/script/core/QxExtend.js
===================================================================
RCS file: /cvsroot/qooxdoo/qooxdoo/source/script/core/QxExtend.js,v
retrieving revision 1.14
diff -u -r1.14 QxExtend.js
--- source/script/core/QxExtend.js	17 Feb 2006 20:42:44 -0000	1.14
+++ source/script/core/QxExtend.js	2 Apr 2006 14:53:13 -0000
@@ -650,11 +650,23 @@
   // building user configured get alias for property
   if (typeof p.getAlias === QxConst.TYPEOF_STRING) {
     pp[p.getAlias] = pp[QxConst.INTERNAL_GET + p.method];
+  }
+  else if(typeof p.getAlias === QxConst.TYPEOF_OBJECT) {
+    for(i = 0; i < p.getAlias.length; i++)
+    {
+      pp[p.getAlias[i]] = pp[QxConst.INTERNAL_GET + p.method];
+    };
   };
 
   // building user configured set alias for property
   if (typeof p.setAlias === QxConst.TYPEOF_STRING) {
     pp[p.setAlias] = pp[QxConst.INTERNAL_SET + p.method];
+  }
+  else if(typeof p.setAlias === QxConst.TYPEOF_OBJECT) {
+    for(i = 0; i < p.setAlias.length; i++)
+    {
+      pp[p.setAlias[i]] = pp[QxConst.INTERNAL_SET + p.method];
+    };
   };
 };
 
