Index: theme.js
===================================================================
RCS file: /cvsroot/qooxdoo/qooxdoo/source/themes/appearance/default/theme.js,v
retrieving revision 1.4
diff -u -r1.4 theme.js
--- theme.js	17 Feb 2006 20:42:45 -0000	1.4
+++ theme.js	21 Feb 2006 10:58:42 -0000
@@ -1994,3 +1994,221 @@
     });
   }
 });
+
+
+/*
+---------------------------------------------------------------------------
+  SCROLLER
+---------------------------------------------------------------------------
+*/
+
+theme.registerAppearance("scroller",
+{
+  initial : function(vWidget, vTheme)
+  {
+    return {
+      overflow : QxConst.OVERFLOW_VALUE_HIDDEN
+    };
+  }
+});
